.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
       - run:
           name: Selection Strategy Comparison
-          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/selection-strategy-comparison/01-selection-strategies.rmd', clean = TRUE)"
+          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/selection-strategy-comparison/01-selection-strategies.rmd', params = list(neighbors = 2), clean = TRUE)"
```

Comment:
Question does the low number of neighbors make this quick to run?

Reply:
It makes it possible to run without failure in the poly-A set case.